BlueprintUI.podspec

```diff
@@ -7,7 +7,7 @@ Pod::Spec.new do |s|
   s.author       = 'Square'
   s.source       = { :git => 'https://github.com/square/blueprint.git', :tag => s.version }
 
-  s.swift_version = '4.2'
+  s.swift_versions = ['4.2', '5.0']
```

Comment:
Up to you, but I think it'd be fine to make this only be `5.0` instead of both versions (we did that recently with workflow, and since swift 4.2 can build with swift 5.0 pods, it probably better to just switch over)